src/Microsoft.Dnx.Runtime/NuGet/Utility/PathUtility.cs

```diff
@@ -145,6 +211,10 @@ public static string GetPathWithBackSlashes(string path)
 
         public static string GetPathWithDirectorySeparator(string path)
         {
+            if (!RuntimeEnvironmentHelper.IsWindows)
```

Comment:
This is incorrect. This helpers is used in other places to normalize slashes in paths.